src/lib.rs

```diff
@@ -37,6 +38,7 @@ extern crate bitflags;
 mod boot_loader_name;
 mod command_line;
 mod elf_sections;
+mod error;
```

Comment:
This doesn't really need to be in a whole module, and leaving it closer to the function it's produced by helps readability imo